### PR TITLE
Log Viewer Improvements

### LINKED ISSUE #540: Log Viewer Improvements
- Move fullscreen button to the left to add a breathing, it feels cluttered just over the vertical scrollbar
- Switch which accordion is open when click previous or next (right now the user doesn't exactly know for which build the log is being displayed)

### PR DESCRIPTION
**Pull request by Francisco2002**
- increase fullscreen button margin left
- add a highlight in select item show in log viewer

Close #540

### CODE REVIEW

File: dashboard/src/components/BootsTable/BootsTable.tsx [hunk 1]
```diff
@@ -150,7 +151,46 @@ const TableCellComponent = ({
...
+  row: Row<TestByCommitHash>;
+  index: number;
+  currentLog?: number;
+  getRowLink: (testId: TestHistory['id']) => LinkProps;
+  openLogSheet: (index: number) => void;
+}): JSX.Element => {
+  const className = useMemo(() => {
+    let initialClassName = 'cursor-pointer hover:bg-lightBlue';
+
+    if (typeof currentLog === 'number' && index === currentLog)
```
Comment (lfjnascimento) [inline dashboard/src/components/BootsTable/BootsTable.tsx:170]: we don't need to do `typeof currentLog === 'number'` all the time, if index is a number `index === undefined` will be false

File: dashboard/src/components/BootsTable/BootsTable.tsx [hunk 2]
```diff
@@ -150,7 +151,46 @@ const TableCellComponent = ({
...
+      initialClassName += ' bg-lightBlue';
```
Comment (lfjnascimento) [inline dashboard/src/components/BootsTable/BootsTable.tsx:171]: nit: just to follow the patter you can use `cn` from utlils

File: dashboard/src/components/BuildsTable/BuildsTable.tsx [hunk 3]
```diff
@@ -95,8 +96,67 @@ const AccordionBuildContentComponent = ({
...
+  row: Row<AccordionItemBuilds>;
+  currentLog?: number;
+  openLogSheet: (index: number) => void;
+  groupHeaders: Header<AccordionItemBuilds, unknown>[];
+  onClickShowBuild: IAccordionItems['onClickShowBuild'];
+}): JSX.Element => {
+  const className = useMemo(() => {
+    let initialClassName = 'cursor-pointer hover:bg-lightBlue';
+
+    if (typeof currentLog === 'number' && index === currentLog)
```
Comment (lfjnascimento) [inline dashboard/src/components/BuildsTable/BuildsTable.tsx:117]: ditto about `typeof currentLog === 'number'`

File: dashboard/src/components/BuildsTable/BuildsTable.tsx [hunk 4]
```diff
@@ -95,8 +96,67 @@ const AccordionBuildContentComponent = ({
...
+      >
+        {row.getVisibleCells().map((cell, cellIdx) => (
+          <TableCellMemoized
+            key={cellIdx}
+            cell={cell}
+            rowIndex={index}
+            row={row}
+            openLogSheet={openLogSheet}
+          />
+        ))}
```
Comment (lfjnascimento) [inline dashboard/src/components/BuildsTable/BuildsTable.tsx:137]: Memoize this array

File: dashboard/src/components/BuildsTable/BuildsTable.tsx [hunk 5]
```diff
@@ -246,55 +306,22 @@ export function BuildsTable({
...
+        modelRows.map((row, index) => {
+          return (
+            <TableRowMemoized
+              key={index}
+              index={index}
+              row={row}
+              groupHeaders={groupHeaders}
+              onClickShowBuild={onClickShowBuild}
+              openLogSheet={openLogSheet}
+              currentLog={currentLog ?? undefined}
```
Comment (lfjnascimento) [inline dashboard/src/components/BuildsTable/BuildsTable.tsx:321]: ditto about currentLog

File: dashboard/src/components/BootsTable/BootsTable.tsx [hunk 6]
```diff
@@ -150,7 +152,47 @@ const TableCellComponent = ({
...
+  row: Row<TestByCommitHash>;
+  index: number;
+  currentLog?: number;
+  getRowLink: (testId: TestHistory['id']) => LinkProps;
+  openLogSheet: (index: number) => void;
+}): JSX.Element => {
+  const className = useMemo(
+    () => (index === currentLog ? 'bg-lightBlue' : undefined),
+    [index, currentLog],
+  );
```
Comment (WilsonNet) [inline dashboard/src/components/BootsTable/BootsTable.tsx:171]: we don't need `useMemo` for this

memo are for expensive calculations and unstable references

an equality comparisson is a preach cheap operation and the result is a primitive which javascript compares by equality

**Review by lfjnascimento — APPROVED** [review]
LGTM

File: dashboard/src/components/BootsTable/BootsTable.tsx [hunk 7]
```diff
@@ -150,7 +152,43 @@ const TableCellComponent = ({
...
+  openLogSheet,
+}: {
+  row: Row<TestByCommitHash>;
+  index: number;
+  currentLog?: number;
+  getRowLink: (testId: TestHistory['id']) => LinkProps;
+  openLogSheet: (index: number) => void;
+}): JSX.Element => {
+  const className = index === currentLog ? 'bg-lightBlue' : undefined;
+  const rowVisibleCells = useMemo(() => row.getVisibleCells(), [row]);
```
Comment (lfjnascimento) [inline dashboard/src/components/BootsTable/BootsTable.tsx:169]: we don't need this useMemo also

Comment (lfjnascimento) [reply dashboard/src/components/BootsTable/BootsTable.tsx:169]: this because `row.getVisibleCells` return a stable reference

**Review by WilsonNet — APPROVED** [review]
nice

**Pull request merged**